pages/positions/consulting-software-engineer.md

```diff
@@ -27,8 +27,14 @@ supervisory_status: 'No'
 ---
 
 {% if page.state == 'upcoming' %}
-{{ page.org }} will soon be accepting applications for a GS-{{ page.gs_level }} {{ page.title }}.
-{% if page.opens == 'tbd' %} The target date for when this position will be officially open to application has not yet been determined. If you'd like to be notified when this position is open, sign up to our [mailing list](https://public.govdelivery.com/accounts/USGSATTS/subscriber/new?topic_id=USGSATTS_6). {% else %} Applications will be open for submission on {{ page.opens | date: '%l:%M%P %Z on %A, %B %e, %Y' }}. Check out [Join TTS Hiring Process]({{site.baseurl}}/hiring-process/) to learn more about the application process. 
+  {{ page.org }} will soon be accepting applications for a GS-{{ page.gs_level }} {{ page.title }}.
+  {% if page.opens == 'tbd' %} The target date for when this position will be officially open to application has not yet been determined. If you'd like to be notified when this position is open, sign up to our [mailing list](https://goo.gl/forms/QMbyPse8f4rfnZ9z2).
+  {% else %} Applications will be open for submission on {{ page.opens | date: '%l:%M%P %Z on %A, %B %e, %Y' }}. Check out [Join TTS Hiring Process]({{site.baseurl}}/hiring-process/) to learn more about the application process.
+  {% endif %}
+{% endif %}
+
+{% if page.state != 'upcoming' %}
```

Comment:
When I was reviewing your changes I also realized that these if statements can just be used to insert things. I think I was also stuck on the idea that it was more like branching logic so all paths had to be defined.

Reply:
Right!  You don't have to follow every "if" with any other conditions or an "else".  You can just use a quick `if`/`endif` statement to, maybe, display a banner when the post if live.